### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by Mwapsam**
- Add Desktop Version

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -363,3 +371,405 @@ form button {
   border: 1px solid #cfd8dc;
   margin-bottom: 45px;
 }
+
+@media screen and (min-width: 800px) {
```
Comment (Nuri1977) [inline main.css:375]: Please change your min-width to 992px for desktop computers. Use media queries in your CSS file to make your desktop version look like the template you selected (let's assume that the desktop width should be in the 992px - 1100px range).

Comment (Nuri1977) [inline main.css:385]: Use media queries in your CSS file to make your desktop version look like the template you selected (let's assume that the desktop width should be in the 992px - 1100px range).

File: index.html [hunk 2]
```diff
@@ -69,107 +77,124 @@ <h1 class="intro-text">I’m Mwape Glad to see you!</h1>
...
+              sign-ups required.
+            </p>
+            <ul class="tech-list">
+              <li><span>html</span></li>
+              <li><span>css</span></li>
+              <li><span>JavaScript</span></li>
+            </ul>
+            <button class="see-project" type="button">See Project</button>
+          </div>
         </div>
```
Comment (Tobinchilongo) [inline index.html:104]: Hi Mwape,
Please make sure that the `div`s are not empty.

File: index.html [hunk 3]
```diff
@@ -69,107 +77,124 @@ <h1 class="intro-text">I’m Mwape Glad to see you!</h1>
...
       </section>
+
       <section class="projects">
         <div class="project-card">
-          <div class="screenshot-project">
-            <img src="assets/project2.png" alt="" />
+          <div class="screenshot-project order-2@md">
+            <img src="assets/cardi-pic.png" alt="" />
```
Comment (ismailakinkunmi) [inline index.html:170]: I will suggest you always add text to alt in img  element, this text will show if the image did not load.

**Review by ismailakinkunmi — APPROVED** [review]

**Pull request closed without merging**